### PR TITLE
Fix Hue integration test

### PR DESCRIPTION
**Pull request by medb**
Failure:
```
======================================================================
FAIL: test_hue.mode: STANDARD.version: 1_0.random_prefix: mbxg (hue.test_hue.HueTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/google/home/idv/.local/lib/python3.6/site-packages/parameterized/parameterized.py", line 518, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/usr/local/google/home/idv/dev/src/dataproc-initialization-actions/hue/test_hue.py", line 51, in test_hue
    machine_suffix
  File "/usr/local/google/home/idv/dev/src/dataproc-initialization-actions/hue/test_hue.py", line 20, in verify_instance
    self.__run_test_file(name)
  File "/usr/local/google/home/idv/dev/src/dataproc-initialization-actions/hue/test_hue.py", line 29, in __run_test_file
    self.assertEqual(ret_code, 0, "Failed to run test file. Error: {}".format(stderr))
AssertionError: 1 != 0 : Failed to run test file. Error: Pseudo-terminal will not be allocated because stdin is not a terminal.
Traceback (most recent call last):
  File "verify_hue_running.py", line 63, in <module>
    main()
  File "verify_hue_running.py", line 56, in main
    assert code is not None, "NOK - Failed to get a status code"
AssertionError: NOK - Failed to get a status code
```

### CODE REVIEW

File: hue/hue.sh [hunk 1]
```diff
@@ -14,15 +14,14 @@
 #
 # This script installs HUE on a master node within a Google Cloud Dataproc cluster.
 
-set -x -e
+set -Eeuxo pipefail
```
Comment (functicons) [inline hue/hue.sh]: Delete E.

File: hue/verify_hue_running.py [hunk 2]
```diff
@@ -52,12 +61,13 @@ def main():
     hue = Hue(BASE, PORT)
     is_main_master = hue.is_main_master()
     if is_main_master:
-        code = hue.get_response_code()
+        code = hue.get_response_code(retries=100)
         assert code is not None, "NOK - Failed to get a status code"
```
Comment (functicons) [inline hue/verify_hue_running.py]: Retry 100 times? seems to be too large.

File: hue/hue.sh (hunk 1 above)
Comment (medb) [reply hue/hue.sh]: Done.

File: hue/verify_hue_running.py (hunk 2 above)
Comment (medb) [reply hue/verify_hue_running.py]: With 3 seconds delay, it just 5 minutes.

**Pull request merged**